### PR TITLE
add user feature FAT

### PR DESCRIPTION
**Pull request by jjiwooLim**
FAT tests for #14677

### CODE REVIEW

**Comment by jjiwooLim** [comment]
#build

**Comment by jjiwooLim** [comment]
#libby-bot

**Comment by jjiwooLim** [comment]
#build

**Review by bensonlatibm — APPROVED** [review]
Looks good!

File: dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/props/PropertiesUtils.java [hunk 1]
```diff
@@ -1,5 +1,9 @@
 /*******************************************************************************
+<<<<<<< HEAD
+ * Copyright (c) 2020, 2021 IBM Corporation and others.
+=======
  * Copyright (c) 2020 IBM Corporation and others.
+>>>>>>> 5862398631b9f73db9d0aa84056e994716190bee
  * All rights reserved. This program and the accompanying materials
```
Comment (Azquelt) [inline dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/props/PropertiesUtils.java:7]: Remove the merge markers from this header.

File: dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/FeatureUtilityToolTest.java [hunk 2]
```diff
@@ -1,5 +1,9 @@
 /*******************************************************************************
+<<<<<<< HEAD
+ * Copyright (c) 2019, 2021 IBM Corporation and others.
+=======
  * Copyright (c) 2019, 2020, 2021 IBM Corporation and others.
+>>>>>>> 5862398631b9f73db9d0aa84056e994716190bee
  * All rights reserved. This program and the accompanying materials
```
Comment (Azquelt) [inline dev/com.ibm.ws.install.featureUtility_fat/fat/src/com/ibm/ws/install/featureUtility/fat/FeatureUtilityToolTest.java:7]: Remove the merge markers

**Review by Azquelt — COMMENTED** [review]
This PR has been reverted. However, when you re-deliver this code, please make sure to fix these headers which still have merge conflict markers in them.

**Pull request merged**